CHANGELOG.md

```diff
@@ -4,6 +4,45 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.0.0-rc1 (2020-10-06)
+
+This release adds major new features and bugfixes since the 1.7.4 release.
+We deem it moderate priority for upgrading.
+
+This release adds the long-awaited support for distributed hypertables to
+TimescaleDB. This release also adds support for user-defined actions allowing
+users to define actions that are run by the TimescaleDB automation framework.
+
+In addition to these major new features, the 2.0 branch introduces _breaking_ changes
+to APIs and existing features, such as continuous aggregates. These changes are not
+backwards compatible and might require changes to clients and/or scripts that rely on
+the previous APIs. Please review our updated documentation and do proper testing to
+ensure compatibility with your existing applications.
+This release candidate is upgradable, so if you are on a previous release (e.g., 1.7.4)
```

Comment:
It might be good to be more specific about API changes. How about?:
```suggestion
ensure compatibility with your existing applications.

The noticeable breaking changes in APIs are:
- Function rename and arguments redesign for compression and retention policies;
- Creating continuous aggregate is split into 2 steps: `CREATE MATERIALIZED VIEW` to create a continuous aggregate view, and `add_continuous_aggregate_policy` to create the actual policy;
- Rename and argument redesign for functions to manipulate running policies;
- Replacing several information views with policy information with general views, which covers policies and user-defined actions;
- Redesign of information views.

This release candidate is upgradable, so if you are on a previous release (e.g., 1.7.4)
```

WDYT?